src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs

```diff
@@ -165,13 +171,14 @@ public async Task<ActionResult> ManageAatfDetails(AatfEditDetailsViewModel viewM
                 viewModel.SiteAddress.Countries = await client.SendAsync(accessToken, new GetCountries(false));
             }
 
-            SetBreadcrumb();
             return View(viewModel);
         }
 
         [HttpGet]
-        public async Task<ActionResult> ManageContactDetails(Guid id)
+        public async Task<ActionResult> ManageContactDetails(Guid id, string type)
```

Comment:
Is it possible to pass the FacilityType enum as action parameter rather than string? Would safe having to convert it